components/CreateChallenge/FormJoin02GroupConfirmation.tsx

```diff
@@ -32,32 +33,22 @@ import LocalPushNotificationSetting from '../LocalPushNotificationSetting';
 
 function FormJoin02GroupConfirmation({navigation, route}, props) {
   const state = useStore(state => state);
-  const [test, setTest] = useState(0);
-  const taskQuantityArray = [];
-  // const taskName = state.challengeInput.taskName;
+  const [loading, setLoading] = useState(true);
```

Comment:
Do you have any reasons why you have to use useState instead of useStore?

Reply:
because loading state is only for this components, I don't want to store it on state.

Reply:
Ok. I merged it because time is limited.